.github/actions/npm-release/action.yml

```diff
@@ -25,9 +25,16 @@ runs:
       run: |
         git config --global user.email "${{ inputs.git_email }}"
         git config --global user.name "${{ inputs.git_username }}"
-    - name: 'Bump version and commit'
+    - name: 'Bump version'
       shell: bash
-      run: npm version ${{ inputs.releaseType }} -m '[skip ci] Publish release %s'
+      run: npm version ${{ inputs.releaseType }} --no-git-tag-version
+    - name: 'Commit and tag version changes'
```

Comment:
These changes plus the addition of the `version` script in the `package.json` file are needed to update `src/version.json` with the new version number prior to the commit/tag. Even though the `npm version` command automatically invokes the `npm run version` script for us, it doesn't happen until _after_ the automatic commit. So we need to tell npm not to do the auto commit with the `--no-git-tag-version` flag and handle it ourselves.